### ISSUE TITLE
Peers behind not can not communicate with each other

### ISSUE THREAD

**Issue by ddfoster0001** [peervpn/peervpn#26]
I have one public server (ps1) and two hosts behind NAT h1 and h2. 

ps1 (Ubuntu 16.04) <--> Public Static IP
h1 (Ubuntu 16.04) <--> Home Router <-->Broadband Internet
h2 (Raspberry Pi 2) <--> USB tethering from Android 4G

I can ping from ps1 (10.10.10.1/24) to both h1 (10.10.10.2/24) and h2 (10.10.10.3/24). And also both h1 and h2 can ping ps1.

But, h1 can't ping h2. Only, ps1 is showing 2 peers connected but h1 and h2 is showing 1 peers connected.

**Comment by conrad784** [comment]
did you try enabling `Option: enablerelay` on ps1?

**Comment by ddfoster0001** [comment]
Following suggestion from @conrad784, I have tried the "enablerelay yes" configuration on ps1. The result is given in the screenshot. Now both h1 and h2 is showing 2 peers connected and can ping each other.

But, I actually need to reach h1 from h2 in the shortest possible route without having to relay through ps1. Ping delay statistic-

1. h1<-->ps1 avg: 281 ms
2. h2<-->ps1 avg: 301 ms
3. h1<-->h2 avg: 560 ms
4. h2<--> Public IP of h1's NAT router avg: 3.6 ms

If h1 could reach h2 by the shortest path, the delay between h1 and h2 would be much less.

![screenshot from 2018-02-22 21-40-25](https://user-images.githubusercontent.com/18414216/36548208-e51b07c8-1819-11e8-9bd3-edd09ea2b71c.png)

**Comment by strfry** [comment]
This is basically a duplicate of #24 .
As far as i know, @tobiasvolk is currently planning a complete rewrite of PeerVPN, meanwhile not much is going to happen here.
Maybe meanwhile in MeshVPN: https://github.com/Kuebler-IT/MeshVPN/issues/3

**Comment by ddfoster0001** [comment]
No, this is not a duplicate of #24. In #24, peers are located under the same NAT but here the peers are under different NAT.

**Comment by davidchan-1908** [comment]
@ddfoster0001  hi, i have the same question with yours,  it make me crary ,could you tell me  how  you handled ? 
Thank you very much！
